src/libpgexporter/queries.c

```diff
@@ -963,7 +963,8 @@ pgexporter_detect_extensions(int server)
       }
       else
       {
-         config->servers[server].extensions[extension_idx].enabled = true;
+         config->servers[server].extensions[extension_idx].enabled =
+            pgexporter_extension_is_enabled(config, server, config->servers[server].extensions[extension_idx].name);
```

Comment:
You're not using the returned value anywhere? And these functions don't seem to have any side-effects as well, so this function call doesn't look like it's doing anything helpful.

Also, weird indentation. Make sure to use `uncrustify` if not done already. Just set it up as a git hook ([pre-commit](https://pre-commit.com/)).